source/SIL.AppBuilder.Portal.Frontend/src/translations/index.tsx

```diff
@@ -14,7 +14,10 @@ i18n
   // https://github.com/i18next/i18next-browser-languageDetector
   .use(LanguageDetector)
   .use(initReactI18next)
-  .use(XHR)
+  // .use(XHR)
```

Comment:
I get from the comment that we want to do local and remote translations.  Double checking that it is ok to turn off the remote translations until the plugin is implemented?

Reply:
at the moment we aren't using any remote translations, but it's planned for 'eventually'